### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by diegoyon**
### List Structure

- Setup project with webpack
- Created index.html file using the webpack plugin.
- Created an array of objects and dynamically display them
- Styled the To Do List to match the styles of the template.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+    description: 'buy food',
+    bool: false,
+    index: 2,
+  },
+  {
+    description: 'send email',
+    bool: false,
+    index: 3,
+  },
+];
```
Comment (zieeco) [inline src/index.js]: - Please, kindly use `completed` instead of `bool` to meet the project requirement.

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
-# todolist
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books
+
```
Comment (zieeco) [inline README.md]: - Please, kindly update the project name to be the name of your current project to make your readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @diegoyon, 



Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights :zap: 

- Nice Design :fire: 
- Descriptive PR :heavy_check_mark: 
- Good commit messages :+1: 
- Linters are passing :100`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — APPROVED** [review]
Hi @diegoyon,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion
 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**